src/collect-using.rst

```diff
@@ -23,6 +23,7 @@ To use ODK Collect:
   collect-forms
   collect-filling-forms
   collect-settings
+  collect-import-export
```

Comment:
It was originally part of the admin settings portion of collect-settings, and I could put it back there, but it seemed way out of proportion to the rest of the information on that page. Also, because it's a specific thing that someone is likely to want to do, I thought it made more sense as its own page.

OR ---

Are you suggesting yes, keep it as its own page, but put the TOC that includes it into the admin setting section? That could work, I think.

Reply:
I think it's fine as its own page, but I think it should go under the Admin Settings in the TOC.